.github/workflows/pkgci_test_sharktank.yml

```diff
@@ -32,8 +32,8 @@ jobs:
 
           - name: rocm_hip
             target: target_hip
-            gpu: gfx1100
-            runs-on: nodai-amdgpu-w7900-x86-64
+            gpu: gfx942
+            runs-on: linux-mi300-2gpu-ossci-iree-org
```

Comment:
How about keeping the existing w7900 job and adding a new mi300 job instead of swapping here? You could even add the new job in a separate PR.

We should be testing on as many configurations as possible:

* CPU
* 1 consumer GPU
* 2 consumer GPUs
* 1 datacenter GPU
* 2 datacenter GPUs

Reply:
Ok is w7900 consumer and mi300 datacenter right?

Reply:
Close enough in this case. AMD PRO W7900 is targeted at professionals and not for gaming, ideally we'd also test on gaming cards like AMD RX 9070. Both are RDNA cards while mi300 is a datacenter CDNA card.